amgut/lib/data_access/sql_connection.py

```diff
@@ -53,6 +53,7 @@ def __init__(self):
         self._connection = None
         self._post_commit_funcs = []
         self._post_rollback_funcs = []
+        self.auto_rollback = False
```

Comment:
I disagree with this change. It looks it is added just because is needed in the test that you're adding below. Instead of modifying this library, that should be the same used in Qiita, you can use the `tearDown` function in the tests. Having small modifications like this in code that is similar (or the same) across projects will make code maintenance harder.